org.alienarena.alienarena.yaml

```diff
@@ -0,0 +1,35 @@
+id: org.alienarena.alienarena
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: /app/src/alienarena/aa/source/bin/alienarena
+modules:
+  - name: alienarena
+    buildsystem: simple
+    no-autogen: true
+    sources:
+      - type: svn
+        url: svn://svn.icculus.org/alienarena/tags/7.71.4
+    build-commands:
+      - ./configure --prefix=/app/src/alienarena/aa/source
+      - make -j$(nproc)
+      - make install
+      - mkdir -p /app/share/icons/hicolor/256x256/apps/
+      - mkdir -p /app/share/applications
+
+      - echo -e "[Desktop Entry]\nName=Alien Arena\nComment=A standalone 3D first person online deathmatch shooter\nComment[es]=Un juego de acción en primera persona sobre red\nComment[it]=Sparatutto online 3D in prima persona\nComment[de]=Ein Internet- und netzwerkfähiger 3D Ego-Shooter\nExec=/app/src/alienarena/aa/source/bin/alienarena\nIcon=org.alienarena.alienarena\nTerminal=false\nType=Application\nCategories=Game;ArcadeGame;\nKeywords=aliens;fps;shooter;multiplayer;" > ./org.alienarena.alienarena.desktop
+    post-install:
+      - install -Dm644 unix_dist/alien-arena.png /app/share/icons/hicolor/256x256/apps/org.alienarena.alienarena.png
+      - install -Dm644 org.alienarena.alienarena.desktop /app/share/applications/
+    build-options:
+        cflags: "-I/app/include"
+    runtime: org.freedesktop.Platform.GL.default
+
+finish-args:
+- --share=ipc
+- --share=network
+- --socket=x11
+- --socket=wayland
```

Comment:
if it supports wayland then line 31 is `fallback-x11`. Otherwise remove line 32.